js/parts/Options.js

```diff
@@ -4,6 +4,14 @@
  * License: www.highcharts.com/license
  */
 
+/**
+ * @typedef {"plot"|"chart"} Highcharts.ButtonAlignValue
+ */
+
+/**
+ * @typedef {"plotBox"|"spacingBox"} Highcharts.ButtonAlign2Value
```

Comment:
Shouldn't it be `ButtonAlignToValue`? And should we try to combine `ButtonAlignValue` with `ButtonAlign2Value`?

Reply:
I fixed it with the merge.